src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -530,7 +530,7 @@ def get_dmatrix(data_path, content_type, csv_weights=0, is_pipe=False):
             files_path = "/tmp/sagemaker_xgboost_input_data"
             shutil.rmtree(files_path, ignore_errors=True)
             os.mkdir(files_path)
-            for path in data_path:
```

Comment:
`data_path` is used as input in the logic branches above as well. Should they also take in a deduped `data_path`?

Reply:
This change is not working so going to change the logic.